client/client_test.go

```diff
@@ -174,17 +174,16 @@ func (s *ClientSuite) addRemoteTarget(c *C, name string) {
 	s.syncRemote(c)
 }
 
-func (s *ClientSuite) rootKeys(c *C) []*data.PublicKey {
-	rootKeys, err := s.repo.RootKeys()
+func (s *ClientSuite) rootMeta(c *C) []byte {
+	meta, err := s.repo.RawMeta("root.json")
```

Comment:
do we really need this method? tbh i would enjoy access to the store through the Repo, but otherwise client suite has access to the store.

would like to avoid adding unnecessary methods for test-only. in a separate PR i would wonder if its warranted to access raw metadata from the Repo struct though.

Reply:
Yeah, I don't like test only code in general. However, I think this is actually a useful thing to have: you're quite often going to want to pull the `root.json` bytes out of the repository to stuff in your client repo or whatever. (maybe just a specific `RawRoot()` would be better?)

WDYT?

Reply:
>  However, I think this is actually a useful thing to have: you're quite often going to want to pull the root.json bytes out of the repository to stuff in your client repo or whatever.

I completely agree! I think though that it's better to access through the `Store` though. Through the `GetMeta()`.

Reply:
If we do go this route, I would prefer `Repo.GetMeta()` that calls the underlying `r.store.GetMeta()`?

Reply:
Oh I think I see what you're saying...1 sec

Reply:
Look now?

Reply:
Nice, I'm happy with that change. It simplifies code for me a lot!

I hated having to hold on to a `Store` AND `Repo` object.

anyone else who is reviewing I hope agrees :)